qiskit_alice_bob_provider/remote/job.py

```diff
@@ -232,7 +232,6 @@ def result(
         status = self.status()
         assert self._last_response is not None
         success = status == JobStatus.DONE
-        print()  # Print a new line to display the next status.
```

Comment:
This was replaced by the more proper `display_new_line` function, to call at the beginning of the call chain